fog/sql_recovery_db/src/lib.rs

```diff
@@ -943,11 +943,11 @@ impl SqlRecoveryDb {
     /// Arguments:
     /// * ingress_key: The ingress key we need ETxOutRecords from
     /// * block_index: The first block we need ETxOutRecords from
-    /// * block_count: How many subsequent blocks to also request data for.
+    /// * block_count: How many consecutive blocks to also request data for.
     ///
     /// Returns:
-    /// * The sequence of ETxOutRecord's, from consequecutive blocks starting
-    ///   from block_index. Empty if not even the block_index'th block exists.
+    /// * The sequence of ETxOutRecord's, from consecutive blocks starting from
```

Comment:
If we're at nit territory, is `starting at block_index` more correct? 🤷

Reply:
I am not sure, but I have no problem to change this if you think that's better